src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -426,6 +382,109 @@ public Task DropSubscriptionConnection()
 
             return NoContent();
         }
+
+        [RavenAction("/databases/*/subscriptions/update", "POST", AuthorizationStatus.ValidUser)]
+        public async Task Update()
```

Comment:
What I think would be nice is to have the ability to create new subscription when it does not exist. Thoughts?

Reply:
I am not sure we want to mix it. 
Now the user can catch SubscriptionDoesNotExistException and send a Create command if he want a update/create

If we make it update/create by default then we will lack the only update scenario, ie when I want to just update existing subscription, it will create me a new one in case it doesn't exist, which seems odd.

Reply:
I'm thinking more about the usability, like we do have with indexes. I'm just doing .Execute and do not worry if the index is there or not.

Do we have .Exists method at least?

Reply:
Note that updating the subscription is something that you should do with some thinking about the current change vector.
Are you resetting it or preserving it?

Reply:
there is no exists method.

Reply:
I think we can add Overwrite option to SubscriptionUpdateOptions ?

Reply:
> Note that updating the subscription is something that you should do with some thinking about the current change vector.
> Are you resetting it or preserving it?

It obey the value of `SubscriptionCreationOptions.ChangeVector`, see test:
https://github.com/ravendb/ravendb/pull/10479/files#diff-2a118f7ce3e7b0101c6c2c0894fefaeaR433

Reply:
Yes, we can add override property there.